CMakeLists.txt

```diff
@@ -27,3 +27,4 @@ add_subdirectory(subsys)
 add_subdirectory(modules)
 add_subdirectory(drivers)
 add_subdirectory(tests)
+add_subdirectory(core_fw)
```

Comment:
I'm not sure if `core_fw` is the best location for this application. My understanding is that `core_fw` is intended to keep applications being maintained by Nordic and hardly modifiable by application developers. While the `remote_radio` application is to be maintained by the application developer.

I understand that customers often treat the network core program as one maintained by Nordic and don't modify it, but we should be open to these customers who would like to introduce some changes.

Reply:
I understood that `core_fw` should be a directory with all sort of "side" images for applications ie. radio images for Bluetooth, or `remote_shell` for UART. That these images should be added automatically as needed like use and forget. That does not exclude the possibility to edit them when necessary.

Moving the `remote` applications to a separate directory allows to distinguish them from the `main` applications without making them unreadable or opaque.

The name `core_fw` in my opinion means "Firmware for another core", but maybe more descriptive name would be more applicable if this is bad.

Reply:
I think we should first define `core_fw` and what kind of firmware goes there. Also, how to enable editing `remote_radio`.

I'm not sure who is responsible for defining the directory structure in sdk-nrf repository. @carlescufi ?